client/src/components/Login.js

```diff
@@ -69,22 +70,15 @@ const Login = () => {
   };
 
   return (
-    <div className="container row col-md-6 mt-5 mx-auto">
-      <h2 style={{ fontSize: "2.2em" }} className="text-center text-light">
-        LOGIN
-      </h2>
-      <Form
-        noValidate
-        validated={validated}
-        onSubmit={handleChange}
-        style={{ width: "50%", margin: "auto" }}
-      >
-        <Form.Group as={Col} className="mb-4" controlId="formBasicEmail">
-          <Form.Label className="text-light">Email</Form.Label>
+    <div className="container">
+      <h1>LOGIN</h1>
```

Comment:
could this be using the heading component? Same as the register page please.

Reply:
Done